rts/motoko-rts/src/idl.rs

```diff
@@ -935,8 +949,8 @@ pub(crate) unsafe fn sub(
     t2: i32,
 ) -> bool {
     if t1 >= 0 && t2 >= 0 {
-        let t1 = t1 as u32;
-        let t2 = t2 as u32;
+        let t1 = t1 as usize;
+        let t2 = t2 as usize;
         if rel.visited(p, t1, t2) {
```

Comment:
Since t1 and t2 are just u32 indices or negative i32 tags, perhaps its just as easy have BitRel use u32 size and indices too (not usize). I.e. there no need to generalize here since Candid is inherently 32-bit.

Reply:
I agree, I put it on my `u32` downgrade backlog.